solidity/ecdsa/.gitignore

```
@@ -2,5 +2,6 @@
 build/
 cache/
 export/
+external/
```

Let's exclude `external/npm` directory as it contains the auto-generated content. We may want to add e.g. `external/mainnet` directory with hardcoded mainnet contracts to use.

```suggestion
external/npm
```